src/filebrowser/listing.ts

```diff
@@ -1180,6 +1180,15 @@ class DirListing extends Widget {
   }
 
   /**
+   * Select a given item by name.
+   */
+  public selectItemByName(name: string): Promise<void> {
```

Comment:
This approach is not going to work because the new item in the listing is created asynchronously.